### PR TITLE
swarm: code cleanup. move to ethersphere/swarm

### PR DESCRIPTION
**Pull request by skylenet**
Removing the swarm codebase from `go-ethereum` 

New home is under [ethersphere/swarm](https://github.com/ethersphere/swarm)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -224,12 +224,3 @@ matrix:
         submodules: false # avoid cloning ethereum/tests
       script:
         - go run build/ci.go purge -store gethstore/builds -days 14
-
```
Comment (karalabe) [inline .travis.yml]: Please remove all the `repo = ethereum/go-ethereum AND ` stuff from the Travis rules that were added to support ethersphere.

File: build/deb/ethereum/deb.control [hunk 2]
```diff
@@ -11,8 +11,8 @@ Vcs-Browser: https://github.com/ethereum/go-ethereum
 Package: {{.Name}}
 Architecture: any
 Depends: ${misc:Depends}, {{.ExeList}}
-Description: Meta-package to install geth, swarm, and other tools
- Meta-package to install geth, swarm and other tools
+Description: Meta-package to install geth, swarm and other tools
```
Comment (karalabe) [inline build/deb/ethereum/deb.control]: Please remove `, swarm`

File: build/deb/ethereum/deb.control [hunk 3]
```diff
@@ -11,8 +11,8 @@ Vcs-Browser: https://github.com/ethereum/go-ethereum
...
+ Meta-package to install geth, and other tools
```
Comment (karalabe) [inline build/deb/ethereum/deb.control]: Please drop the comma after geth

File: build/deb/ethereum/deb.control (hunk 2 above)
Comment (nonsense) [reply build/deb/ethereum/deb.control]: I think `ethereum-swarm` is still part of the `ethereum` metapackage, no?

Not sure if we should remove it, but if we are keeping the current behaviour (i.e. installing geth, swarm, etc. if you install `ethereum`), then this should stay I think.

Comment (nonsense) [reply build/deb/ethereum/deb.control]: On a second though, it seems that we have removed `ethereum-swarm` from the meta package, so actually @karalabe is correct :) `ethereum-swarm` is no longer installed as part of `ethereum`, so we should remove all references of `swarm`.

**Comment by nonsense** [comment]
@karalabe has anyone optimized the running time of the go-ethereum test suite more than @skylenet ;)

**Comment by karalabe** [comment]
@nonsense :trollface:

**Review by kaycoinsUSA — COMMENTED** [review]
Amazing

**Pull request merged**